chain/network/src/types.rs

```diff
@@ -769,6 +773,20 @@ pub enum PeerMessage {
     EpochSyncResponse(EpochSyncResponse),
     EpochSyncFinalizationRequest(EpochId),
     EpochSyncFinalizationResponse(EpochSyncFinalizationResponse),
+
+    RoutingTableSyncV2(RoutingSyncV2),
```

Comment:
It might make sense to make this a versioned data structure as well in case we want to change it again in the future

Reply:
I added `version` field